rules/rules_test.go

```diff
@@ -563,23 +563,23 @@ func TestExpressions(t *testing.T) {
 		{
 			expr: `round(-1 * (1 + 0.005 * http_requests{group="production",job="api-server"}))`,
 			output: []string{
-				`{group="production", instance="0", job="api-server"} => -2 @[%v]`,
+				`{group="production", instance="0", job="api-server"} => -1 @[%v]`,
```

Comment:
An alternative would be to round half away from zero instead. I think it might be a less surprising behaviour.